src/main/java/com/gluonhq/substrate/target/AndroidTargetConfiguration.java

```diff
@@ -296,18 +296,14 @@ boolean allowHttps() {
 
     @Override
     List<String> getTargetSpecificAOTCompileFlags() throws IOException {
-        Path llcPath = getLlcPath();
-        Path internalLlcPath = projectConfiguration.getGraalPath().resolve("lib").resolve("llvm").resolve("bin");
-        
         return Arrays.asList("-H:CompilerBackend=" + Constants.BACKEND_LLVM,
+                "--features=org.graalvm.home.HomeFinderFeature",
```

Comment:
this should go to AbstractTargetConfiguration

Reply:
Should we enable the feature unconditionally or only when LLVM backend is used?

Reply:
unconditionally. At this moment, it is by default enabled in GraalVM for the LIR backend, but defaults can change.